src/IssueViz/IssueViz.csproj

```diff
@@ -30,6 +30,25 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Remove="Commands\Resources\CommandIcons.png" />
+    <None Remove="IssueVisualizationControl\IssueVisualizationControl.xaml" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="Commands\Resources\CommandIcons.png" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.ComponentModelHost" Version="14.0.25424" />
+    <PackageReference Include="Microsoft.VisualStudio.Shell.14.0" Version="14.3.25407" />
+    <PackageReference Include="Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime" Version="16.7.30328.74" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Page Include="IssueVisualizationControl\IssueVisualizationControl.xaml">
+      <Generator>XamlIntelliSenseFileGenerator</Generator>
+      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+    </Page>
```

Comment:
Grouping of related elements: VS seems to make additional/deletions to the project file at arbitrary locations.
The project file is easier to read related entries are grouped e.g.
* all PackageReferences together
* items that are removed and re-added with a different item type.